src/lavinmq/mqtt/session.cr

```diff
@@ -43,8 +43,10 @@ module LavinMQ
           Fiber.yield if (i &+= 1) % 32768 == 0
         rescue ::IO::Error
           @log.error { "deliver loop exited due to IO error" }
```

Comment:
This may be expected, so dunno if we should log this as error? Or use different levels depending on `@closed`?

Reply:
feel like it should at least be warn, bc it also might be unexpected 
or do you think it can go down to an info?